### DISCUSSION TITLE
Ideal payment url

### DISCUSSION THREAD

**Discussion by NeatFastro** [flutter-stripe/flutter_stripe#338] [Q&A]
Can we change the ideal payment url right now it's showing "r.girogate.de" but the device is in Netherlands.
Also how can i change the default country of payment sheet

**Comment by remonh87** [comment]
The default country is selected based on the localisation of the Phone. At the moment there is nothing we can change this since Stripe needs to fix it. 

About Girogate I think Stripe is using this I would ask them about this since we do not set such an url. Maybe also check: https://stripe.com/docs/payments/ideal/accept-a-payment?platform=react-native
